src/slices/loginSlice.ts

```diff
@@ -0,0 +1,62 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export type loginUser = {
+  email: string;
+  password: string;
+};
+
+type InitialState = {
+  loading: boolean;
+  userInfo: Object;
+  token: string;
+  error: string;
+  success: string;
+};
+
+const initialState: InitialState = {
+  loading: false,
+  userInfo: {},
+  token: "",
+  error: "",
+  success: "",
+};
+
+const LOGIN_URL = "http://localhost:5000/login";
```

Comment:
better to use env variable with .env and reuse with something like {process.env. API_URL}